deflect/qt/QmlStreamerImpl.cpp

```diff
@@ -178,9 +180,9 @@ void QmlStreamer::Impl::_initRenderer()
 {
     _updateSizes( QSize( _rootItem->width(), _rootItem->height( )));
 
-    _quickRenderer = new QuickRenderer( *_quickWindow, *_renderControl,
-                                        true );
+    _quickRenderer = new QuickRenderer( *_quickWindow, *_renderControl, true );
 
+#ifdef DEFLECTQT_MULTITHREADED
 #if QT_VERSION >= 0x050500
```

Comment:
#else here should probably just static_assert(false) instead of printing a warning because it does not work at all